From: SeongJae Park <sj@kernel.org>
Date: Sun, 26 Oct 2025 10:20:49 -0700
Subject: [PATCH] ==== docs fixups ====

mm/damon: misc documentation fixups

As the subject says.

First three patches fix up issues in the documents, including wrong
explanation of a behavior, wrong link, and a contextual typo.  Following
five patches update documents for not yet documented features and
behaviors.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_k56_k0pc | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_k56_k0pc

diff --git a/hkml_cv_bogus/hkml_cv_bogus_k56_k0pc b/hkml_cv_bogus/hkml_cv_bogus_k56_k0pc
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.47.3

